### ISSUE TITLE
PEP 541: transfer ownership of unmaintained `PyTeaser` package

### ISSUE THREAD

**Issue by verhovsky** [pypa/warehouse#6413]
I tried emailing @xiaoxu193 back in January (7 months ago) and got no response. My Python 3 support pull request has been open since that time as well and there's no activity on Github. 

I'd like to take over maintenance of this package.

**Comment by jamadden** [comment]
As a moderator, I've attempted to contact the user via GitHub and email. The email address we have on file has not been verified and there hasn't been any activity on GitHub or PyPI from the corresponding accounts in some time.

**Comment by xiaoxu193** [comment]
Hello, if you would like, I can add you as a collaborator to this project on Github: https://github.com/xiaoxu193/PyTeaser

**Comment by verhovsky** [comment]
Yes please. Can you also let me push updates to PyPI please? https://pypi.org/user/verhovsky
